src/ripple/app/main/Amendments.cpp

```diff
@@ -57,7 +57,8 @@ supportedAmendments ()
         { "DC9CA96AEA1DCF83E527D1AFC916EFAF5D27388ECA4060A88817C1238CAEE0BF EnforceInvariants" },
         { "3012E8230864E95A58C60FD61430D7E1B4D3353195F2981DC12B0C7C0950FFAC FlowCross" },
         { "CC5ABAE4F3EC92E94A59B1908C2BE82D2228B6485C00AFF8F22DF930D89C194E SortedDirectories" },
-        { "B4D44CC3111ADD964E846FC57760C8B50FFCD5A82C86A72756F6B058DDDF96AD fix1201" }
+        { "B4D44CC3111ADD964E846FC57760C8B50FFCD5A82C86A72756F6B058DDDF96AD fix1201" },
+        { "6E3FF89E20D93A6C51FAE78409EB68B74DCEA0FA8AA50A2A364630241DC18976 fix1523" }
```

Comment:
`B9E739B8296B4A1BB29BE990B17D66E21B62A300A909F25AC55C22D6C72E1F9D` for `fix1523`?

Reply:
Oops. Last minute rename!